sdk/tables/azure-data-tables/azure/data/tables/_error.py

```diff
@@ -51,15 +50,8 @@ def _wrap_exception(ex, desired_type):
     msg = ""
     if len(ex.args) > 0:
         msg = ex.args[0]
-    if sys.version_info >= (3,):
-        # Automatic chaining in Python 3 means we keep the trace
-        return desired_type(msg)
-
-    # There isn't a good solution in 2 for keeping the stack trace
-    # in general, or that will not result in an error in 3
-    # However, we can keep the previous error type and message
+    return desired_type(msg)
```

Comment:
Where is the `_wrap_exception` function used?

Reply:
In _authentication.py: https://github.com/Azure/azure-sdk-for-python/blob/a1c902e000ddc0a20e64ed9bf51bcdf9a6bb255c/sdk/tables/azure-data-tables/azure/data/tables/_authentication.py#L151